eng/common/scripts/Verify-Links.ps1

```diff
@@ -145,6 +145,10 @@ function CheckLink ([System.Uri]$linkUri)
     catch {
       $statusCode = $_.Exception.Response.StatusCode.value__
 
+      if(!$statusCode){
```

Comment:
nit: add extra space before `{`

Reply:
Can you also add a comment here that the ErrorCode is common when there is a SocketExeception and so we are try to pull the ErrorCode from the inner SocketException if it exists.

Reply:
```suggestion
      if(!$statusCode) {
```